library/ix-dev/community/vaultwarden/questions.yaml

```diff
@@ -29,6 +29,16 @@ portals:
     path: "$kubernetes-resource_configmap_portal_admin_path"
 
 questions:
+  - variable: TZ
```

Comment:
To be sure - this will automatically get picked up by common so it reflects this in the container?